frontend/src/TrackFile/FileDetails.js

```diff
@@ -0,0 +1,210 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+import formatTimeSpan from 'Utilities/Date/formatTimeSpan';
+import Link from 'Components/Link/Link';
+import DescriptionList from 'Components/DescriptionList/DescriptionList';
+import DescriptionListItem from 'Components/DescriptionList/DescriptionListItem';
+import DescriptionListItemTitle from 'Components/DescriptionList/DescriptionListItemTitle';
+import DescriptionListItemDescription from 'Components/DescriptionList/DescriptionListItemDescription';
+import styles from './FileDetails.css';
+
+function FileDetails(props) {
+
+  function renderRejections() {
+    const {
+      rejections
```

Comment:
One issue on SonarCloud UI project (doesn't show up in checks). Because you have function embedded in another function you shouldn't have to re-declare the rejections variable, move the variable declaration below to above the renderRejections function or move the renderRejections out of the main function.